scipy/sparse/csgraph/__init__.py

```diff
@@ -43,32 +43,61 @@
 Graph Representations
 =====================
 This module uses graphs which are stored in a matrix format.  A
-graph with N nodes can be represented by an (N x N) adjacency matrix G.
-If there is a connection from node i to node j, then G[i, j] = w, where
-w is the weight of the connection.  For nodes i and j which are
+graph with ``N`` nodes can be represented by an ``(N x N)`` adjacency matrix ``G``.
+If there is a connection from node ``i`` to node ``j``, then ``G[i, j] = w``, where
+``w`` is the weight of the connection.  For nodes ``i`` and ``j`` which are
 not connected, the value depends on the representation:
 
-- for dense array representations, non-edges are represented by
-  G[i, j] = 0, infinity, or NaN.
+- for *dense array* representations, non-edges are represented by
+  ``G[i, j] = 0``, infinity, or NaN.
 
-- for dense masked representations (of type np.ma.MaskedArray), non-edges
-  are represented by masked values.  This can be useful when graphs with
-  zero-weight edges are desired.
+- for *dense masked* representations (of type `numpy.ma.MaskedArray <http://docs.scipy.org/doc/numpy/reference/maskedarray.generic.html#what-is-a-masked-array>`_),
```

Comment:
This should also work without the full link, i.e. `` `numpy.ma.MaskedArray` ``) I think.